### ISSUE TITLE
PDF Permissions Password

### ISSUE THREAD

**Issue by marceloueda** [illsk1lls/ZipRipper#51]
Is it possible to use it to recover Permissions Password instead of Document Open Password?

**Comment by illsk1lls** [comment]
Probably not but give it a shot and let us know. The way to add permission passwords and file open passwords are the same, and both say "Encrypt the file." So there is a possibility.

**Comment by marceloueda** [comment]
It seems that it would need some tweaking. It detects and process it as an encrypted file and it says a password was found, but the password list is blank.

**Comment by illsk1lls** [comment]
obviously you tried entering a blank password and that didnt work correct?

I'll ask about it on JtR's page and see what they come up with

**Comment by marceloueda** [comment]
Correct. Actually I tried in a file which I put the password so I already knew it (there was no password to open the file, only a permissions password). Thanks!

**Comment by Mohamed421984** [comment]
CraxsRATعايز افك ابضغط بيطلب كلمة مرور

**Comment by Mohamed421984** [comment]
> CraxsRATaaiz افك ابضغط بيطلب كلمة مرور

ارجو الافادة
